obj.js

```diff
@@ -309,15 +309,19 @@ async function createObjectDescription(obj, id) {
   }
 
   let props = null;
-  for (let propName in obj) {
-    if (propName.startsWith("_") ||
-        typeof(obj[propName]) == "function") {
+  for (let propName of Object.getOwnPropertyNames(obj)) {
+    if (propName.startsWith("_")) {
+      continue;
+    }
+    let property = Object.getOwnPropertyDescriptor(obj, propName);
+    if (property.get ||
+        typeof(property.value) == "function") {
       continue;
     }
     if ( !props) {
       props = {};
     }
-    props[propName] = await mapOutgoingObjects(obj[propName]);
+    props[propName] = await mapOutgoingObjects(property.value);
```

Comment:
What was wrong with the old line?

Reply:
I was just being consistent as I had changed all of the other places where I had used `getOwnPropertyDescriptor`.